frontend/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
```

Comment:
Good point! Fixed the root linter failing in 65b66427 by having it ignore the new files.

I think it might indeed be a good idea to have a `package.json` at the root, but to ensure that we can merge this without breaking the current website, I think I'd prefer doing that _after_ we move over.

I don't quite know yet how we'd do that though. Specifically, I don't know of a way to make it so that `npm install` at the root installs all regular and especially dev dependencies in `/frontend`, and I don't expect the back-end folks to be too happy if I move the contents of `/frontend` one level up, interspersed with the back-end code? @maxxcrawford any ideas/preferences?